### ISSUE TITLE
Cargo install?

### ISSUE THREAD

**Issue by Zolmok** [rapiz1/rathole#90]
> A full-powered rathole can be obtained from the release page. Or build from source for other platforms and minimizing the binary. A Docker image is also available.

Why not `cargo install rathole`?

I tried but it fails
```
error: failed to run custom build command for `rathole v0.3.1`

Caused by:
  process didn't exit successfully: `/tmp/cargo-installeiw1LE/release/build/rathole-4166e51a1e9957bd/build-script-build` (exit status: 1)
  --- stderr
  Error: could not find repository from '/home/hackerzol/.cargo/registry/src/github.com-1ecc6299db9ec823/rathole-0.3.1'; class=Repository (6); code=NotFound (-3)
warning: build failed, waiting for other jobs to finish...
error: failed to compile `rathole v0.3.1`, intermediate artifacts can be found at `/tmp/cargo-installeiw1LE`

Caused by:
  build failed
```

**Comment by rapiz1** [comment]
Sorry. the code on crates.io hasn't been updated for a while. I will work on CI to make crates.io up to date.

I just published v0.3.4 to crates.io. `cargo install rathole` should work now.

**Comment by rapiz1** [comment]
Now CI will automatically publish to crates.io
